packages/shared/src/components/tooltips/SimpleTooltip.tsx

```diff
@@ -29,11 +30,30 @@ export function SimpleTooltip({
     [children],
   );
 
+  const onTrigger = (_, event) => {
+    if (event.type !== 'focus') {
+      shouldShow = true;
+    }
+  };
+
+  const onUntrigger = (_, event) => {
+    const bodyPath = event.path.filter((path) => document.body === path)[0];
+    if (bodyPath.className === 'ReactModal__Body--open') {
+      shouldShow = false;
+      return;
+    }
+    shouldShow = true;
+  };
+
   return (
     <TippyTooltip
       shouldLoad={getShouldLoadTooltip()}
       {...props}
       content={content}
+      onClick
```

Comment:
I think there is nothing declared in this scope as `onClick`.

Reply:
Yep! 
Missed that in removing it, tried to bind to that first.

Will remove it